### ISSUE TITLE
Traceback with command after updating to 3.8.0

### ISSUE THREAD

**Issue by PsychoSid** [datawhores/OF-Scraper#343]
I run the following as part of a shell script to process pre/post scrape. This worked fine for 3.7.x

```
ofscraper -u <comma list of creators> --posts all --action download,like --discord NORMAL --original
```

After updating to 3.8.0 I get:-

```
[paths.cleanup:42]  Cleaning up temp files                                                                                                                                                                                                                        [paths.py](file:///home/seedit4me/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/utils/paths/paths.py):42
                                                                                                                                                                                                                                                                              
                                                                                                                                                                                                                                                                              
 [run.main:29]  Traceback (most recent call last):                                                                                                                                                                                                                   [run.py](file:///home/seedit4me/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/runner/run.py):29
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/commands/scraper.py", line 80, in main                                                                                                                                         
    paths.cleanup()                                                                                                                                                                                                                                                           
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/utils/paths/paths.py", line 44, in cleanup                                                                                                                                     
    data.get_TempDir("audio")                                                                                                                                                                                                                                                 
TypeError: config_reader.<locals>.inner() takes 0 positional arguments but 1 was given                                                                                                                                                                                        
                                                                                                                                                                                                                                                                              
During handling of the above exception, another exception occurred:                                                                                                                                                                                                           
                                                                                                                                                                                                                                                                              
Traceback (most recent call last):                                                                                                                                                                                                                                            
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/runner/run.py", line 17, in main                                                                                                                                               
    main_helper()                                                                                                                                                                                                                                                             
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/runner/run.py", line 47, in main_helper                                                                                                                                        
    picker.pick()                                                                                                                                                                                                                                                             
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/commands/picker.py", line 20, in pick                                                                                                                                          
    scraper.main()                                                                                                                                                                                                                                                            
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/commands/scraper.py", line 98, in main                                                                                                                                         
    paths.cleanup()                                                                                                                                                                                                                                                           
  File "/home/your_username/.local/pipx/venvs/ofscraper/lib/python3.11/site-packages/ofscraper/utils/paths/paths.py", line 44, in cleanup                                                                                                                                     
    data.get_TempDir("audio")                                                                                                                                                                                                                                                 
TypeError: config_reader.<locals>.inner() takes 0 positional arguments but 1 was given                                                                                                                                                                                        
                                                                                                                                                                                                                                                                              
 [run.main:30]  config_reader.<locals>.inner() takes 0 positional arguments but 1 was given
```

Note as part of the script I move the `save_location` directory to a date stamped suffix so it doesn't exist on initial run. Older versions didn't mind but this version fails out if it doesn't exist with the above command. So I added it into the script to create it prior to running.

Many thanks for the work.

**Comment by PsychoSid** [comment]
`"save_location": "/home/seedit4me/ofs"`

**Comment by datawhores** [comment]
This was in the discord and should be fixed with 3.8.1
